pyp5js/tests/http/test_web_app.py

```diff
@@ -0,0 +1,19 @@
+from pyp5js.http.web_app import app as web_app
+from flask_testing import TestCase
+
+class WebAppTests(TestCase):
+
+    render_templates = False
+
+    def create_app(self):
+        app = web_app
+        app.config['TESTING'] = True
+        return app
+
+    def test_get_home_renders_index_template(self):
+        self.client.get('/')
+        self.assert_template_used('index.html')
+
+    def test_get_new_sketch_renders_new_sketch_form_template(self):
+        self.client.get('/new-sketch/')
+        self.assert_template_used('new_sketch_form.html')
```

Comment:
I hope I'm not being to picky, but I think a better organization for the tests would be to have 1 test class per route. It's something really helpful when it comes to preparing data via `setUp` or as url routing. It also improves the readability of the tests as well. What do you think about this type of structure:

```python
class IndexViewTests(Pyp5jsWebTestCase):
    route = '/'

    def test_get_home_renders_index_template(self):
        self.client.get(self.route)
        self.assert_template_used('index.html')


class NewSketchViewTests(Pyp5jsWebTestCase):
    route = '/new-sketch/'

    def test_get_new_sketch_renders_new_sketch_form_template(self):
        self.client.get(self.route)
        self.assert_template_used('new_sketch_form.html')
```

Reply:
@berinhard Sounds good! I haven't really thought about splitting into different test classes but it makes sense grouping the scenarios by route. Will do that!
Some questions:
1. Do you think that will be good to have them in different files (one per class) or is it ok having them all inside `test_web_app.py`? Taking a look on the other tests I saw that there is usually one class per file and some fixtures, but I really don't have strong feelings in this case. Either way is fine for me.
2. For the parent class `Pyp5jsWebTestCase` the idea here is to have a base class for the web app route tests with the boilerplate for setting up the Flask app (the method `def create_app(self):` would be part of this base class for instance)?

Reply:
1 - I don't think we need to bring this level of complexity right now. I mean, this is probably what we'll want to have in the future, but since now we only have a single `web_app.py` file, I think it's ok for the tests to reflect that with a single `tests_web_app.py` file.

2 - Yep. This parent class should have 3 methods and should be responsible for creating the sketchbook_dir (under `setUp`), clearing all created data by removing the same dir (under `tearDown`) and to set up the flask app (under `create_app`).